//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdeg_T_P.h
//
// Code generation for function 'acdeg_T_P'
//

#ifndef ACDEG_T_P_H
#define ACDEG_T_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdeg_T_P(double P_init, double V_init, double A_init, double P_wayp,
               double V_max, double V_min, double J_max, double J_min, double T,
               creal_T t[21]);

#endif
// End of code generation (acdeg_T_P.h)
